arm-apimanagement/2016-10-10/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1270 @@
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of API Management services."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The response of the List API Management services operation."
+    },
+    "ApiManagementServiceGetSsoTokenResult": {
+      "properties": {
+        "redirect_uri": {
+          "type": "string",
+          "description": "Redirect URL containing the SSO token."
+        }
+      },
+      "description": "The response of the GetSsoToken operation."
+    },
+    "ApiManagementServiceCheckNameAvailabilityParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name to check for availability."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Parameters supplied to the CheckNameAvailability operation."
+    },
+    "ApiManagementServiceNameAvailabilityResult": {
+      "properties": {
+        "nameAvailable": {
+          "type": "boolean",
+          "description": "True if the name is available and can be used to create a new API Management service; otherwise false."
+        },
+        "message": {
+          "type": "string",
+          "description": "If reason == invalid, provide the user with the reason why the given name is invalid, and provide the resource naming requirements so that the user can select a valid name. If reason == AlreadyExists, explain that <resourceName> is already in use, and direct them to select a different name."
+        },
+        "reason": {
+          "type": "string",
+          "description": "Invalid indicates the name provided does not match the resource provider’s naming requirements (incorrect length, unsupported characters, etc.)  AlreadyExists indicates that the name is already in use and is therefore unavailable.",
+          "enum": [
+            "Valid",
+            "Invalid",
+            "AlreadyExists"
+          ],
+          "x-ms-enum": {
+            "name": "NameAvailabilityReason",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "nameAvailable"
+      ],
+      "description": "Response of the CheckNameAvailability operation."
+    },
+    "ApiManagementServiceUploadCertificateParameters": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Hostname type.",
+          "enum": [
+            "Proxy",
+            "Portal",
+            "Management",
+            "Scm"
+          ],
+          "x-ms-enum": {
+            "name": "HostnameType",
+            "modelAsString": false
+          }
+        },
+        "certificate": {
+          "type": "string",
+          "description": "Base64 Encoded certificate."
+        },
+        "certificate_password": {
```

Comment:
We will take this as feedback and update it in next version of the API. This api-version is already being used, we are just updating the swagger for documentation purpose.

---
In reply to: [106763958](https://github.com/Azure/azure-rest-api-specs/pull/1039#discussion_r106763958) [](ancestors = 106763958)